benches/nexmark/config.rs

```diff
@@ -37,16 +22,89 @@ pub struct Config {
     pub auction_proportion: usize,
 
     #[clap(
-        long = "bit-proportion",
+        long = "bid-proportion",
```

Comment:
```suggestion
        long,
```
Clap will rename it automatically, so this can be applied to other members too (e.g. `auction_proportion`)